packages/manager/modules/sign-up/src/form/details/details.controller.js

```diff
@@ -32,6 +32,29 @@ export default class SignUpDetailsCtrl {
         },
       },
     };
+
+    this.zipModel = {
+      value: null,
+      model: (...args) => {
```

Comment:
```
model: (zipModel) => {
  if (zipModel) {
    ...
```
?

Reply:
`zipModel` can be `null` and then never be setted